### PR TITLE
feat: add parachain destination fees

### PR DESCRIPTION
**Pull request by Victor-Salomon**
Add parachain destination fee to Registry + integrate

### CODE REVIEW

File: app/src/config/registry.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ export namespace Mainnet {
     logoURI: 'https://s2.coinmarketcap.com/static/img/coins/64x64/22125.png',
     chainId: 3369,
     network: Network.Polkadot,
+    destinationFeeDOT: 500_000_000,
```
Comment (noahjoeris) [inline app/src/config/registry.ts:38]:
```suggestion
    destinationFeeDOT: 500_000_000n,
```

This way no?

File: app/src/hooks/useTransfer.tsx [hunk 2]
```diff
@@ -158,7 +158,9 @@ const useTransfer = () => {
           token.address,
           destinationChain.chainId,
           amount,
-          500_000_000n,
+          destinationChain.destinationFeeDOT
+            ? BigInt(destinationChain.destinationFeeDOT)
+            : BigInt(0),
```
Comment (noahjoeris) [inline app/src/hooks/useTransfer.tsx:163]:
```suggestion
          destinationChain.destinationFeeDOT || 0n,
```

File: app/src/models/chain.ts [hunk 3]
```diff
@@ -13,6 +13,8 @@ export interface Chain {
   chainId: number
   /** Network the chain belongs to, e.g., Polkadot, Ethereum. */
   network: Network
+  /** Optionnal parameter for parachain only */
```
Comment (noahjoeris) [inline app/src/models/chain.ts:16]:
```suggestion
  /** Optional parameter for parachains only. Used for xcm transfers */
```

**Review by noahjoeris — CHANGES_REQUESTED** [review]
left some comments

File: app/src/config/registry.ts (hunk 1 above)
Comment (Victor-Salomon) [reply app/src/config/registry.ts:38]: I actually did this on purpose. It's gonna be stored on the local storage and it does not handle BN properly. This value should just be used in the useTransfer hook which expects a BN and wield if you try to provide a number. This is why I've decided to go this way

**Pull request merged**